scripts/4.covariates.py

```diff
@@ -83,34 +76,15 @@
 # 
 # Another useful covariate will be the logarithm plus one function of the number mutations that was calculated in the aforementioned notebook.
 
-# In[9]:
-
-path = os.path.join('data', 'mutation-matrix.tsv.bz2')
-mutation_df = pd.read_table(path, index_col=0)
-
-
-# In[10]:
-
-mutation_df.head(2)
-
-
-# In[11]:
-
-number_of_mutations = mutation_df.sum(axis='columns')
-number_of_mutations.head(2)
-
-
-# Let's add this as an additional column to our covariates.
-
-# In[12]:
+# In[8]:
 
-covariates_df['n_mutations_log1p'] = np.log1p(number_of_mutations)
-covariates_df.head()
+covariates_df['n_mutations_log1p'] = np.log1p(covariates_df.pop('n_mutations'))
```

Comment:
I removed it because I envisioned `covariates.tsv` being used for modeling (otherwise you would want `samples.tsv` without the dummies). For modeling, I think you will want the `log1p`.